pkg/unit_tests/runtime.go

```diff
@@ -0,0 +1,123 @@
+package unit_tests
+
+import (
+	"net"
+	"os"
+	"path/filepath"
+
+	"github.com/rancher/k3s/pkg/daemons/config"
+	"github.com/rancher/k3s/pkg/daemons/control/deps"
+	"github.com/rancher/k3s/pkg/version"
+)
+
+// All certs and other files are stored in /tmp/k3s/<RANDOM_STRING>/
```

Comment:
Can you update the doc comments to be in the expected format? https://golang.org/doc/effective_go#commentary

Reply:
I have updated all function comments to conform to the expected format